Miniforge3/construct.yaml

```diff
@@ -12,5 +12,6 @@ license_file: ../LICENSE
 
 specs:
   - python >=3,<3.8
+  - tqdm
```

Comment:
Is this needed? If so, it should be added as a dependency to conda

Reply:
I know I am trying to debug the aarch64 installer.